CODEOWNERS

```diff
@@ -33,7 +33,7 @@ Wallet/                         @akegalj @flyingleafe
 txp/                            @gromakovsky @pva701 @neongreen
 update/                         @gromakovsky @pva701
 lrc/                            @pva701
-godtossing/                     @neongreen
+ssc/                            @neongreen
```

Comment:
I want to share ownership of `ssc/` :)
Can you add me in the next part?

Reply:
Okay.